src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -18,13 +24,32 @@
      * @throws ParseException if the user input does not conform the expected format
      */
     public DeleteCommand parse(String args) throws ParseException {
-        try {
-            Index index = ParserUtil.parseIndex(args);
-            return new DeleteCommand(index);
-        } catch (IllegalValueException ive) {
-            throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteCommand.MESSAGE_USAGE));
+        args = args.trim();
+        List<String> indexStrs = Arrays.asList(args.split(" "));
+        //eliminate duplicates
+        HashSet<Integer> indexIntsSet = new HashSet<>();
+
+        for (String indexStr : indexStrs) {
+            try {
+                indexIntsSet.add(ParserUtil.parseInt(indexStr));
+            } catch (IllegalValueException e) {
+                throw new ParseException(
+                        String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteCommand.MESSAGE_USAGE));
+            }
         }
+
+        List<Integer> indexInts = new ArrayList<>(indexIntsSet);
```

Comment:
Can sort the ArrayList<Index> instead of converting it to integer just to sort :p